src/loaddb/load_db.c

```diff
@@ -190,35 +186,12 @@ ldr_check_file_name_and_line_no (load_args * args)
 	    }
 	  if (*q == 0)
 	    {
-	      schema_file_start_line = atoi (p + 1);
-	      *p = 0;
+	      return atoi (p + 1);
 	    }
 	}
     }
 
-  if (!args->index_file.empty ())
-    {
-      /* *INDENT-OFF* */
-      p = strchr (const_cast<char *> (args->index_file.c_str ()), ':');
-      /* *INDENT-ON* */
-      if (p != NULL)
-	{
-	  for (q = p + 1; *q; q++)
-	    {
-	      if (!char_isdigit (*q))
-		{
-		  break;
-		}
-	    }
-	  if (*q == 0)
-	    {
-	      index_file_start_line = atoi (p + 1);
-	      *p = 0;
-	    }
-	}
-    }
-
-  return 0;
+  return 1;
```

Comment:
`return 1` looks very confusing. please clarify its meaning by: a) explanatory comment, b) output argument start_line_no instead of return, c) other option